src/types/types.ts

```diff
@@ -351,3 +351,35 @@ export interface SavedRpcMethod extends LocalStorageSavedItem {
   shareableUrl: string | undefined;
   payload: AnyObject;
 }
+
+// =============================================================================
+// Smart Contract Explorer
+// =============================================================================
+export type ContractInfoApiResponse = {
+  contract: string;
+  created: number;
+  creator: string;
+  account?: string;
+  payments?: number;
+  trades?: number;
+  wasm?: string;
+  storage_entries?: number;
+  validation?: {
+    status?: "verified" | "unverified";
```

Comment:
I only see `status`, `ts`, `repository`, and `commit` from `ContractValidationStatus` from the api:
```status:
          description: Validation status
          type: string
          enum: [ verified,unverified ]
        ts:
          description: Validation timestamp
          $ref: '#/components/schemas/Timestamp'
        repository:
          type: string
          description: GitHub repository URL
          example: 'https://github.com/reflector-network/reflector-contract'
        commit:
          type: string
          description: Commit hash
          example: '391f37e39a849ddf7543a5d7f1488e055811cb68'```

Reply:
Same as above, it's in the API response. Thanks for checking! :raised_hands: